easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+local_pytorch_ver = '1.4.0'
+versionsuffix = '-Python-%%(pyver)s-PyTorch-%s' % local_pytorch_ver
```

Minor suggestion: I think that we can drop the variable `local_pytorch_ver` as it is not used anywhere
```suggestion
versionsuffix = '-Python-%(pyver)s-PyTorch-1.4.0'
```